<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Sample</class>
 <widget class="QWidget" name="Sample">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>599</width>
    <height>434</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QPlainTextEdit" name="plainTextEdit">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>GroupBox</string>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="2" column="0" colspan="2">
         <widget class="QPushButton" name="checkPattern">
          <property name="text">
           <string>Check</string>
          </property>
         </widget>
        </item>
        <item row="5" column="2" colspan="2">
         <widget class="QPushButton" name="delField">
          <property name="text">
           <string>Delete</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="labelField">
          <property name="text">
           <string>Field:</string>
          </property>
         </widget>
        </item>
        <item row="5" column="4">
         <widget class="QPushButton" name="performParse">
          <property name="text">
           <string>Perform</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="labelPattern">
          <property name="text">
           <string>Pattern:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QLabel" name="labelGroup">
          <property name="text">
           <string>Group:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="3" colspan="2">
         <widget class="QSpinBox" name="spinGroup"/>
        </item>
        <item row="3" column="1" colspan="4">
         <widget class="QLineEdit" name="textField"/>
        </item>
        <item row="0" column="1" colspan="4">
         <widget class="QLineEdit" name="textPattern"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="labelMatch">
          <property name="text">
           <string>Match:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1" colspan="4">
         <widget class="QLineEdit" name="textMatch"/>
        </item>
        <item row="5" column="0" colspan="2">
         <widget class="QPushButton" name="addField">
          <property name="text">
           <string>Add</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QTableWidget" name="tableFields">
       <column>
        <property name="text">
         <string>Field</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Pattern</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Group</string>
        </property>
       </column>
      </widget>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QPushButton" name="loadFields">
         <property name="text">
          <string>Load</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="saveFields">
         <property name="text">
          <string>Save</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
